zeppelin/converter.py

```diff
@@ -4,17 +4,18 @@
 import os
 from datetime import datetime
 
-"""
-ZeppelinConverter is a utility to convert Zeppelin raw json into Markdown
-
-"""
-
 
 class ZeppelinConverter:
 
+    """
+    ZeppelinConverter is a utility to convert Zeppelin raw json into Markdown
+
```

Comment:
Please strip out unnecessary blank lines in all the docstrings in this file. A blank line is necessary only when you need to separate the first line (short description) from the rest. And if you have multiple paragraphs in the long  description of course :)